joomla/cassiopeia
# Blog Sample Data - New custom fields, tags

### Summary of Changes
This PR adds new data to the content in blog sample data.

Tags
Fieldgroup
Field 

All loadLanguage statements come before the exception is thrown.

Modules are moved to position bottom-b


### Testing Instructions
Install blog sample data in a fresh installation.

In the backend you now must have 
a Field Group
a Field in this field group. 
four tags

In the frontend:
Check that articles have the new Tab "The Autor"
For two articles (Workflow and About your homepage) the author fields have some content.
In the frontend, the author - field must be shown below the text.

Tags are shown for articles in category blog

![grafik](https://user-images.githubusercontent.com/1035262/97818536-2a4f0200-1ca3-11eb-95c1-67a331bc8ac7.png)



### Documentation Changes Required
If there is a documentation about sample data: yes
